### ISSUE TITLE
Gybe Asmz - transport controls of multiple selected layers

### ISSUE THREAD

**Issue by gybeasmz** [photopea/photopea#740]
Previous issue for moving multiple layers has been partialy resolved, now I´m able to move multiple layers without the need of selecting them with ctrl shift T. But selected layers still don´t have visualization, that they are selected. Only the first selected layers has transport controls (the little rectangles in corners). It´s not apparent which layers are selected.

**Comment by photopea** [comment]
What do you mean? Do you mean Transform Controls (enabled in the top menu)? They should visualize a rectangle of all selected layers.

**Comment by gybeasmz** [comment]
yeah, sorry, that was a typo. But let me rephrase my issue. When I draw 3 rectangles and click on one of them, then hold shift and click the other 2, nothing happens. The other 2 rectangles are selected, but there is no indication what so ever on the canvas part. In the layer list, I can see them selected, but on the canvas they look untouched.

**Comment by photopea** [comment]
I think I know what you mean. It should be fixed now.

**Comment by gybeasmz** [comment]
now it´s broken totally. I can´t  select another layer and I can even drag layer and not holding a mouse over it.

**Comment by photopea** [comment]
Can you be more precise? What did you do? It does not happen for me.

**Comment by gybeasmz** [comment]
ok I open new project, I draw 3 rectangles. I switch to move tool and I can´t 
1) click anything - nothing happens when I click any of those 3 rectangles
2) select multiple layers - because nothing happens no matter what I do.

**Comment by photopea** [comment]
I made a new project and drew 3 rectangles (with Rectangle Tool). Then I switched to the Move tool.

I am able to move the last rectangle. When I enable Auto-Select, I can also move other rectangles.

**Comment by photopea** [comment]
You have to enable Paths (View - Paths) to see contours around selected vector shapes.

The contours are thin. You can enable Distances in the top panel to see what is selected.

**Comment by photopea** [comment]
Without Auto-select, you always move the active layer, no matter where you click. It is more convenient for many people (they select layers in Layers panel, because they have a nice structure).
